src/Sentry/MetricBucketHelper.cs

```diff
@@ -0,0 +1,25 @@
+namespace Sentry;
+
+internal static class MetricBucketHelper
+{
+    private const int RollupInSeconds = 10;
+
+    private static readonly DateTime EpochStart = new(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
There's a `DateTime.UnixEpoch`, can we use that?

https://learn.microsoft.com/en-us/dotnet/api/system.datetime.unixepoch?view=net-8.0